### PR TITLE
Add `by_alias` parameter to `avro_schema`

### PR DESCRIPTION
**Pull request by nikitabarskov**
So we can get a schema based on the fields of pydantic class as based on the aliases of fields.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pydantic-avro"
-version = "0.3.1"
+version = "0.3.2"
```
Comment (ffinfo) [inline pyproject.toml:3]: Please revert this, I should do a new release for this

File: src/pydantic_avro/base.py [hunk 2]
```diff
@@ -7,9 +7,9 @@ class AvroBase(BaseModel):
     """This is base pydantic class that will add some methods"""
 
     @classmethod
-    def avro_schema(cls) -> dict:
+    def avro_schema(cls, by_alias: bool = True) -> dict:
         """Return the avro schema for the pydantic class"""
```
Comment (ffinfo) [inline src/pydantic_avro/base.py:11]: Can this be added to the doc string?

**Review by ffinfo — CHANGES_REQUESTED** [review]
Just some small things but this seems like a awesome change

File: pyproject.toml (hunk 1 above)
Comment (nikitabarskov) [reply pyproject.toml]: Fixed.

File: src/pydantic_avro/base.py (hunk 2 above)
Comment (nikitabarskov) [reply src/pydantic_avro/base.py]: Please check that the format of docstrings I used is correct. I used the same as pydantic is using. But looks like fixed.

Comment (ffinfo) [reply src/pydantic_avro/base.py]: That all fine :)

**Comment by ffinfo** [comment]
will make a new release for this

**Comment by ffinfo** [comment]
version 0.3.2 is released

**Pull request merged**